pkg/aspect/root/flags/bazel_flags.go

```diff
@@ -0,0 +1,95 @@
+/*
+Copyright © 2021 Aspect Build Systems Inc
+
+Not licensed for re-use.
+*/
+
+package flags
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"aspect.build/cli/pkg/bazel"
+)
+
+var (
+	exposedBazelFlags = []string{
+		"keep_going",
+		"expunge",
+		"expunge_async",
+		"show_make_env",
+	}
+)
+
+type MultiString struct {
+	value *[]string
+}
+
+func (s *MultiString) Set(value string) error {
+	*s.value = append(*s.value, value)
+	return nil
+}
+
+func (s *MultiString) Type() string {
+	return "multiString"
+}
+
+func (s *MultiString) String() string {
+	return fmt.Sprintf("[ %s ]", strings.Join(*s.value, ", "))
+}
+
+func (s *MultiString) First() string {
+	return (*s.value)[0]
+}
+
+func AddBazelFlags(cmd *cobra.Command) error {
+	subCommands := make(map[string]*cobra.Command)
+
+	for _, command := range cmd.Commands() {
+		subCommands[command.Use] = command
+	}
+
+	bzl := bazel.New()
+	if bzlFlags, err := bzl.Flags(); err != nil {
+		return fmt.Errorf("unable to determine available bazel flags: %w", err)
+	} else {
```

Comment:
Since the `if` is just an error-check + return this doesn't need to be in an `else`?

Reply:
Refactored